docs/Development-Guides/Rate-Limits.md

```diff
@@ -29,3 +22,166 @@ Product | Max requests initiated per minute
 `employment` | 4
 `payment` | 2
 `pay-statement` | 2
+
+## Application Rate Limits
+
+Similarly, multiple access tokens can be created from a single Finch application as more employers are connected. A Finch application has its own rate limits separate from the access token rate limits. You can think of an application like a development environment or application stage. Finch will assign three types of applications: `sandbox`, `development`, and `production`. Each will have their own `client_id` which represents the application.
+
+For application-level rate limits, each `product` manages another “larger” bucket simultaneously counting all requests across all access tokens created by that application. Each bucket is still scoped to a product endpoint like with access tokens; the only difference is that the application-level bucket is larger.
+
+Product | Max requests initiated per minute
+-------|-------------
+`company` | 20
+`directory` | 20
+`individual` | 20
+`employment` | 20
+`payment` | 12
+`pay-statement` | 12
+
+## IP Address Rate Limits
+
+Finch also enforces individual IP Address rate limits. An IP address is a unique address that identifies a device on the internet or a local network. If the number of `max requests` are sent from the same IP Address within the `duration` time set, a `penalty` is enforced. No more requests are allowed once the penalty is enforced. Once the penalty duration is complete, requests will be accepted again.
+
+Type | Max requests | Duration | Penalty
+-------|-------------|-------|-------------
+`API` | 1000 | 5 minutes | 60 minutes
+
+## Other types of rate limits
+
+### Finch Connect Rate limits
+
+Finch enforces rate limits on Employers going thru [Finch Connect](/docs/Integrating-with-Finch/Integrate-Finch-Connect/Overview.md). Each "step" in Finch Connect (the provider selection, logging in, multi-factor authentication prompts, etc) makes a single request to the Finch `/auth/authorize` endpoint. It is highly unlikely for an Employer to encounter these rate limit under normal use. However, an example of this rate limit being encountered is if an employer forgot their password and tried 30 different passwords in the time period of 1 minute. In this case, they would receive a rate limit penalty of 5 minutes.
```

Comment:
Our Finch Connect React app (in Netlify) is the only client of `/auth/authorize` -- an external developer would never call this directly. It would make sense to omit this detail here (just adds confusion).

Reply:
Removed